crates/router/src/connector/stripe/transformers.rs

```diff
@@ -1190,7 +1190,8 @@ impl TryFrom<(&payments::BankRedirectData, Option<bool>)> for StripeBillingAddre
 
 fn get_bank_debit_data(
     bank_debit_data: &payments::BankDebitData,
-) -> (StripePaymentMethodType, BankDebitData, StripeBillingAddress) {
+) -> Result<(StripePaymentMethodType, BankDebitData, StripeBillingAddress), errors::ConnectorError>
+{
```

Comment:
This change is `unnecessary`, you are not throwing any error from the the function

Reply:
Removed